windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -792,13 +792,18 @@ keyboard that you use in Windows.  Keyman keyboards will adapt automatically to
   <!-- Context: Splash Dialog -->
   <!-- String Type: PlainText -->
   <!-- Introduced: 8.0.288.0 -->
-  <string name="S_Splash_Name" comment="Splash major title">Keyman</string>
+  <string name="S_Splash_Name" translatable="false" comment="Splash major title">Keyman</string>
 
   <!-- Context: Splash Dialog -->
   <!-- String Type: PlainText -->
   <!-- Introduced: 8.0.294.0 -->
   <string name="S_Splash_Start" comment="Start Keyman button">Start Keyman</string>
 
+  <!-- Context: Splash Dialog -->
+  <!-- String Type: PlainText -->
+  <!-- Introduced: 18.0 -->
+  <string name="S_Splash_Start_2" comment="Start {Keyman} button">Start %1$s</string>

```

Comment:
> This change will mean all the other languages will, until translated have the english "Start" on the splash screen. We could quite easily do the translation ourselves looking at the existing start splash screen. Should we do that in crowdin?

Maybe we tackle other translations on a separate PR. After this new string gets merged to master, I'll upload to Crowdin first.